src/test/handler.test.ts

```diff
@@ -29,47 +36,103 @@ describe('getParamsFromUriPath', () => {
+        params: {
+            repo: 'sourcegraph/sourcegraph',
+            file: '#web/src/storm/index.tsx',
+        },
+        expected: projects[0],
+    },
+    {
+        goal: 'returns a dev project that matches the repo and file patterns',
+        params: {
+            repo: 'sourcegraph/dev-repo',
+            file: '#dev/backend/main.go',
+        },
+        expected: projects[1],
+    },
+    {
+        goal: 'returns undefined for not matching repo and file patterns',
+        params: {
+            repo: 'sourcegraph/test-repo',
+            file: '#dev/test/start.rb',
+        },
+        expected: undefined,
+    },
+    {
+        goal: 'returns undefined for not matching repo and file patterns',
+        params: {
+            repo: 'sourcegraph/test-repo',
+            file: '#dev/test/start.rb',
+        },
+        expected: undefined,
+    },
+]
 
 describe('matchSentryProject', () => {
-    it('returns a web project that matches the repo and file patterns', () =>
-        expect(matchSentryProject(paramsWeb, projects)).toEqual(projects[0]))
+    beforeEach(async () => {
+        await sourcegraph.configuration.get().update('sentry.organization', 'sourcegraph')
+        await sourcegraph.configuration.get().update('sentry.projects', projects)
+    })
+    for (const paramsCase of paramsInput) {
+        it('fullfils the following goal: ' + paramsCase.goal, () =>
+            expect(matchSentryProject(paramsCase.params, projects)).toEqual(paramsCase.expected)
+        )
+    }
+})
 
-    it('returns a dev project that matches the repo and file patterns', () =>
-        expect(matchSentryProject(paramsDev, projects)).toEqual(projects[1]))
+const incompleteConfigs = [
+    {
+        goal: 'return one missing config',
+        settings: {
+            name: 'sourcegraph',
+            projectId: '1334031',
+            patternProperties: {
+                repoMatches: undefined,
+                fileMatches: [/(web|shared|src).*\.java?/, /(dev|src).*\.java?/, /.java?/],
+                lineMatches: [/logger\.debug\(['"`]([^'"`]+)['"`]\);/],
+            },
+        },
+        expected: ['repoMatches'],
+    },
+    {
+        goal: 'return two missing configs',
+        settings: {
+            name: 'sourcegraph',
+            projectId: '',
+            patternProperties: {
+                repoMatches: undefined,
+                fileMatches: [/(web|shared|src).*\.java?/, /(dev|src).*\.java?/, /.java?/],
+                lineMatches: [/logger\.debug\(['"`]([^'"`]+)['"`]\);/],
+            },
+        },
+        expected: ['projectId', 'repoMatches'],
+    },
+]
 
-    it('returns undefined for not matching repo and file patterns', () =>
-        expect(matchSentryProject(paramsNone, projects)).toEqual(undefined))
+describe('missingConfig', () => {
+    for (const config of incompleteConfigs) {
+        it('check missing configs with goal to ' + config.goal, () =>
```

Comment:
He, she, it, das "s" muss mit 😛 
```suggestion
        it('checks missing configs with goal to ' + config.goal, () =>
```